sdl_android/src/main/java/com/smartdevicelink/encoder/SdlEncoder.java

```diff
@@ -9,11 +9,14 @@
 import android.media.MediaCodecInfo;
 import android.media.MediaFormat;
 import android.os.Build;
+import android.util.Log;
```

Comment:
I forgot to put my question on this.

I see several log outputs in this project. Which one should I use here?
- Log.i()
- DebugTool.logInfo()
- SdlTrace.logProxyEvent()

Reply:
This is a fair question, and I don't have a specific answer. I think we need to write a proposal that revamps the logging feature as it is not clear or unified.